subsys/bluetooth/controller/Kconfig

```diff
@@ -96,7 +96,6 @@ choice BT_LL_CHOICE
 
 config BT_LL_SW_SPLIT
 	bool "Software-based BLE Link Layer"
-	select BT_RECV_IS_RX_THREAD
```

Comment:
don't we want to keep this @cvinayak ? I don't see where the option is selected otherwise.

Reply:
https://github.com/zephyrproject-rtos/zephyr/pull/40273#discussion_r838438179

Reply:
fair enough, approved :)